ios-target-app/Onboarding/Views/SignInViewController.swift

```diff
@@ -98,7 +118,7 @@ class SignInViewController: UIViewController {
         scrollView.attachHorizontally(to: view, leadingMargin: 0, trailingMargin: 0)
         scrollView.attachVertically(to: view, topMargin: 0, bottomMargin: 0)
         
-        stackView.attachVertically(to: scrollView, topMargin: 40, bottomMargin: 0)
+        stackView.attachVertically(to: scrollView, topMargin: UI.StackView.marginBig, bottomMargin: 0)
```

Comment:
this is regarding the previous code review #2 isn't it? I'd recommend to do it in the same to avoid confusions and mistakes. You should merge that PR before this

Reply:
Yeah, I was thinking about that but this PR is pointing to merge with the branch of the previous PR. So, that's why I did those changes in this and merge, and then I will merge the previous one.